### PR TITLE
:tada: start a sub-chapter on personal website framework

### PR DESCRIPTION
**Pull request by jyoti-bhogal**
### Summary

I created a sub-section on ' Frameworks ' in the new personal websites chapter (created by @sayalaruano ) in the communication guide. 

### List of changes proposed in this PR (pull-request)

* First draft of the sub-section for the 'Frameworks` page for the personal websites chapter.
* Added the new sub-section for the chapter file into the _tocml.

### What should a reviewer concentrate their feedback on?

- [ ] Are there any frameworks that you think should be added or removed from this draft?
- [x] Everything looks ok?

### Acknowledging contributors

- [ ] All contributors to this pull request are already named in the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file.
- [ ] The following people should be added to the [table of contributors](https://github.com/the-turing-way/the-turing-way/blob/main/README.md#contributors) in the README file: @sayalaruano, @likeajumprope, and @jyoti-bhogal .

### CODE REVIEW

File: book/website/communication/personal-website/personal-website-frameworks.md [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+
+## Popular frameworks
+
+Below mentioned are some of the commonly used frameworks. These have been
+divided into two categories depending upon whether person wants to create a
+website using low-code or high-code options.
+
+### No-Code/Low-Code frameworks
+
+* [Git hub profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)
```
Comment (SaranjeetKaur) [inline book/website/communication/personal-website/personal-website-frameworks.md:33]:
```suggestion
* [GitHub (GH) profile](https://docs.github.com/en/account-and-profile/setting-up-and-managing-your-github-profile/customizing-your-profile/about-your-profile)
```

Comment (Preshh0) [inline book/website/communication/personal-website/personal-website-frameworks.md:33]: GitHub is typically one word.

**Review by Preshh0 — COMMENTED** [review]
I think this is a great chapter. I have dropped a comment on a correction I think should be made.

Otherwise, this is great! ✨

**Comment by nehamoopen** [comment]
I've used Hugo (https://gohugo.io/) as an alternative to Jekyll. I've found it nicer to work with, `blogdown` for R also uses Hugo as a framework.

The `postcards` package for R is also a nice low-code option (https://github.com/seankross/postcards). It can also be linked to Distill (https://rstudio.github.io/distill/) websites in R, but I think people will be moving to Quarto in place of `blogdown` and `distill`.

**Review by KirstieJane — COMMENTED** [review]
I haven't read the content but I'm sure others will do.

A quick note from me that I don't think it makes sense to merge this into `main` without the additional context from the other sub-chapters that are being created.

So you probably want to create a combined branch to allow @jyoti-bhogal @sayalaruano and @likeajumprope to merge their individual contributions into one combined chapter that can be reviewed as a whole 😄

**Comment by jyoti-bhogal** [comment]
Moved all the commits from this PR to another PR #3937 in order to have all the contributions to the 'Personal Websites' chapter under one branch.

**Pull request closed without merging**